packages/manager/apps/public-cloud/src/components/walkMe/templates/popover.html

```diff
@@ -0,0 +1,14 @@
+<div class="popover tour oui-popover visible">
+    <div class="walkme-line"></div>
+    <button class="oui-button oui-button_icon-alone walkme-close mr-1 pt-2 position-absolute" data-role="end">
+        <span class="oui-icon oui-icon-close_thin"></span>
+    </button>
+    <div class="walkme-title mt-2 position-relative">
+        <h3 class="popover-title py-0 pl-2 border-bottom-0 bg-transparent"></h3>
+        <div class="walkme-bubble ml-3 position-absolute"></div>
+    </div>
+    <div class="popover-content oui-popover__content-container px-4 py-0"></div>
+    <div class="popover-navigation d-flex justify-content-between p-0">
+        {{ navigation }}
```

Comment:
```suggestion
        {{:: navigationText }}
```
?

Reply:
No because it is not only text. `navigationContent` maybe ?

Reply:
One-way binding is useless here as AngularJS won't go through this